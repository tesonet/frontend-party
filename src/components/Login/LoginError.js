import styled from 'styled-components';

export default styled.div`
  color: white;
  font-size: 16px;
  letter-spacing: 0.35px;
  margin: auto;
  margin-top: 10px;
`;
