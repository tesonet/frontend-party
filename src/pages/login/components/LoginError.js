import React from 'react'

export const LoginError = ({ errorMessage }) => {
    return (
        <div>
    		{errorMessage ? (<div>{errorMessage}</div>):('')}
    	</div>
    )
}