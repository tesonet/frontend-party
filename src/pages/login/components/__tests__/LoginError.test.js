import { LoginError } from '../LoginError';

it("should match snapshot", () => {
    const wrapper = shallow(
        <LoginError errorMessage='Error message'/>
    );

    expect(wrapper).toMatchSnapshot();
})
it("should render error message", () => {
    const wrapper = mount(
        <LoginError errorMessage='Error message'/>
    );
    const text = wrapper.text();

    expect(text).toEqual('Error message')
})