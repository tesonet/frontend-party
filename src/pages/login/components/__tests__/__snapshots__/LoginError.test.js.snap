// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="error-message"
>
  <div>
    Error message
  </div>
</div>
`;
